Libplanet.Extensions.Cocona/Libplanet.Extensions.Cocona.csproj

```diff
@@ -11,7 +11,7 @@
 
   <PropertyGroup>
     <LangVersion>8.0</LangVersion>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Since #1138, `Libplanet.Extensions.Cocona` project became to target only `netstandard2.0` because the [`Cocona`](https://www.nuget.org/packages/Cocona/) library targets only `netstandard2.0`. Is there other reason to target only  `netcoreapp3.1` again?

Reply:
It was my misunderstanding.
reverted.